### DISCUSSION TITLE
GUI Recommendations

### DISCUSSION THREAD

**Discussion by XBeg9** [strimzi/strimzi-kafka-operator#4863] [Q&A]
Any recommendations on GUI or tools to ease the management of the Strimzi Kafka cluster?

Currently using https://conduktor.io/, but its price is a bit high for a "personal" project.

**Comment by scholzj** [comment]
It would be great if someone actually using some UI regularly can reply and share their experiences. I personally do not really use any.

But from discussions on Slack, some people seem to be using Kafdrop (https://github.com/obsidiandynamics/kafdrop I guess?). Kowl (https://github.com/cloudhut/kowl) is another UI which looks interesting and @weeco as one of the authors is usually also on the Strimzi Slack channel, so he can help with any issues or questions which is also an advantage.

**Reply by XBeg9** [reply]
Kowl can't control/manage the cluster... :(

**Reply by Yedidyah27** [reply]
Take a look at [https://github.com/yahoo/CMAK](url)
